spec/isolator/ignorer_spec.rb

```diff
@@ -33,7 +33,8 @@ def self.call_unmasked(a, b)
 
   before do
     allow(Isolator).to receive(:within_transaction?) { true }
-    allow(File).to receive(:exist?).with(TODO_PATH).and_return(true)
+    allow(File).to receive(:exist?).and_call_original
+    allow(File).to receive(:exist?).with(todo_path).and_return(true)
```

Comment:
I was running into `uninitialized constant RSpec::Support::Differ` without this.